addons/purchase_stock/models/account_invoice.py

```diff
@@ -168,6 +169,7 @@ def _stock_account_prepare_anglo_saxon_in_lines_vals(self):
                         'analytic_tag_ids': [(6, 0, line.analytic_tag_ids.ids)],
                         'exclude_from_invoice_tab': True,
                         'is_anglo_saxon_line': True,
+                        'ref': move.ref,
```

Comment:
No. This field is supposed to be `readonly`.

Reply:
But it is not. https://github.com/odoo/odoo/blob/13.0/addons/account/models/account_move.py#L2652

Reply:
I’ll check if it’s a bug or not

Reply:
@HviorForgeFlow Confirmed with the team: This field is supposed to be readonly. Sorry :x